src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
```

Comment:
Consider adding a comment or a blank line here to separate the initialization of `sum` from the `adder` function definition. This can improve readability.